client/src/components/History/CurrentHistory/HistoryOperations.vue

```diff
@@ -88,7 +88,7 @@
                         <span v-localize>Delete All Hidden Content</span>
                     </b-dropdown-item>
                     <b-dropdown-item v-b-modal:purge-all-deleted-content>
-                        <span v-localize>Purge All Hidden Content</span>
```

Comment:
I thought that this would purge all hidden datasets, not all deleted datasets?

Reply:
I think this message was probably a copy&paste from the one above, the actual defined function and the `v-b-modal` is `purge-all-deleted-content`.
Also, *deleting all hidden content* seems like an unsafe operation since usually, when you create a collection, the source datasets will be hidden and you may destroy them without being aware.